docker/docker-compose.yml

```diff
@@ -1,18 +1,15 @@
 # Note that mongo container is started before Orion, as Orion needs it
 # as dependency and reserve the order has been found problematic in some
 # low resource hosts
-
-# Uncomment the --smallfiles line if you host doesn't have too much free space
 mongo:
```

Comment:
In addition, maybe the following explanation about the mongo command would be useful in the .md setion:

"Regarding --smallfiles, you can ommit it if your hosting machine have enough disk space. Regarding --nojournal it is not recommened for production, but it speeds up mongo container start up and avoid some "race conditions" problems if Orion containers is faster and doesn't find the DB up and ready"